### PR TITLE
Bumped prod RELEASE for 1.19-5

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by TerryHowe — APPROVED** [review]
/lgtm

File: projects/kubernetes/kubernetes/1-19/KUBE_GIT_VERSION_FILE [hunk 1]
```diff
@@ -1,6 +1,6 @@
-KUBE_GIT_COMMIT='fd5d41537aee486160ad9b5356a9d82363273721'
```
Comment (TerryHowe) [inline projects/kubernetes/kubernetes/1-19/KUBE_GIT_VERSION_FILE:1]: Don't we need this change with the dev release bump?

Comment (kschumy) [reply projects/kubernetes/kubernetes/1-19/KUBE_GIT_VERSION_FILE]: Moved this to https://github.com/aws/eks-distro/pull/394, which is when the dev number gets bumped. Lmk if this isn't ok

Comment (kschumy) [reply projects/kubernetes/kubernetes/1-19/KUBE_GIT_VERSION_FILE]: Also I hard reset the commits back HEAD, which I know isn't the best practices for the team but I didn't want there to be any chance that something weird would happen git-wise with this PR

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Pull request merged**